.github/workflows/check.yaml

```diff
@@ -9,46 +9,32 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            - run: source includes.sh && summarize
+            - run: source includes.sh summarize
 
             - uses: actions/setup-node@v4
               with:
                   cache: yarn
                   node-version-file: .nvmrc
-            - run: npm -g install yarn && yarn
+            - run: yarn install --frozen-lockfile
 
             - uses: actions/setup-python@v5
               with:
                   python-version-file: .python-version
-            - uses: actions/cache@v4
```

Comment:
Why get rid of the caching?

Reply:
There's debate, and experimental data from others, questioning whether caching is actually a speedup.

In the last run of this Action, the bootstrap pip install took 2 seconds and uv pip install requirements took 4 seconds. I think that's fast enough for now.